service/kvmclusterconfig/v1/resource/encryptionkey/desired.go

```diff
@@ -48,6 +48,8 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		},
 	}
 
+	r.logger.LogCtx(ctx, "debug", "desired encryption key secret computed")
```

Comment:
Do I understand you correctly that you want to log the same message in the beginning of the function and in the end of the function? Can you help me understand how it would be useful? In this case it doesn't have any context with it (though now that I think of it, I'll add some) and I think it's useless in the combined logs because you don't know whether operation started or ended or if it actually restarts but somehow fails in the middle.

Reply:
> Do I understand you correctly that you want to log the same message in the beginning of the function and in the end of the function?

No. I realized I made a typo above. I suggest to replace the debug message with what I proposed because the message style is more aligned with the message at the top of the method. So the first and the second message should be like this. 

```
computing desired encryption key secret
computed desired encryption key secret
```

Not like this. 

```
computing desired encryption key secret
desired encryption key secret computed
```

There is a pattern so far and breaking it hurts the little OCD Jesus inside my heart. :D